src/setup/scss/extends/_invisible.scss

```diff
@@ -0,0 +1,8 @@
+%invisible {
+  position: absolute;
```

Comment:
How about using the styles, which are officially recommended to work good in combination with Screenreaders, see https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/